sample/lib/controllers/sample_radio.controller.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:get/get_state_manager/get_state_manager.dart';
+
+/// GetX Controller for holding Radios's current value
+class SampleRadioController extends GetxController {
+  final List<String> optionsString = ["Option 1", "Option 2"];
+
+  String selectedRadio = "Option 1";
```

Comment:
Change all these properties to be observables, so the observer can listen to their updates.

Reply:
Funciona também, usei este exemplo no switch, porém quis fazer assim para  termos outros exemplos de implementação para ilustrar as possibilidades do getx.

Reply:
Done

Reply:
It works too, I used this example in the switch, but I wanted to do so so we could have other implementation examples to illustrate the possibilities of getx.

Reply:
Done